p2p/host/autonat/grafana-dashboards/autonat.json

```diff
@@ -0,0 +1,658 @@
+                                                        }
+                                                ]
+                                        }
+                                },
+                                "overrides": [
+                                        {
+                                                "matcher": {
+                                                        "id": "byName",
+                                                        "options": "rate limit"
+                                                },
+                                                "properties": [
+                                                        {
+                                                                "id": "color",
+                                                                "value": {
+                                                                        "fixedColor": "red",
+                                                                        "mode": "fixed"
+                                                                }
+                                                        }
+                                                ]
+                                        }
+                                ]
+                        },
+                        "gridPos": {
+                                "h": 8,
+                                "w": 12,
+                                "x": 0,
+                                "y": 15
+                        },
+                        "id": 8,
+                        "options": {
+                                "legend": {
+                                        "calcs": [],
+                                        "displayMode": "list",
+                                        "placement": "bottom",
+                                        "showLegend": true
+                                },
+                                "tooltip": {
+                                        "mode": "single",
+                                        "sort": "none"
+                                }
+                        },
+                        "targets": [
+                                {
+                                        "datasource": {
+                                                "type": "prometheus",
+                                                "uid": "${DS_PROMETHEUS}"
+                                        },
+                                        "editorMode": "code",
+                                        "expr": "increase(libp2p_autonat_server_dialrefused_total[$__rate_interval])",
+                                        "legendFormat": "{{refusal_reason}}",
+                                        "range": true,
+                                        "refId": "A"
+                                }
+                        ],
+                        "title": "Server: Dial requests rate limited",
```

Comment:
This doesn't seem to work?

<img width="828" alt="image" src="https://user-images.githubusercontent.com/1478487/218884646-ba93997f-8979-4f34-903a-03857a137b65.png">

Reply:
Just checked my dashboard, and now it's showing one event (after a couple of hours), so it seems to work.

Why is this not part of the dashboard to the right though?

Reply:
The dashboard on the right has segregation by Message_ResponseStatus. So this point would be counted under dial refused. It does seem fine to separate rate limit and dial refused and display on the same dashboard. Should I do that?

Reply:
Added other Refusal reasons too.